### DISCUSSION TITLE
for some reason proxman is not working anymore

### DISCUSSION THREAD

**Discussion by guocity** [ProxymanApp/Proxyman#2127] [General]
the following always appear when proxyman starts, 
proxyman Version 5.8.0 (50800)

<img width="768" alt="image" src="https://github.com/user-attachments/assets/55271bac-2dfe-4f02-9455-594b44e13b3a">

**Comment by NghiaTranUIT** [comment]
It seems the Proxyman Certificate is expired? Or you're using any VPN at the same time 🤔 

You can try to factory reset in the Help menu -> Advance -> Factory Reset: Uncheck the "Include your Data"

**Reply by guocity** [reply]
I have done the reset, reinstall the certificate, I have Cloudflare warp installed but not on

**Reply by NghiaTranUIT** [reply]
May I ask if you install & trust the certificate on your Mac ?

You can find it in the Certificate menu -> Install for Mac

Doc: :https://docs.proxyman.io/debug-devices/macos

**Reply by NghiaTranUIT** [reply]
Does it happen with all website? Or just www.google.com ?

**Reply by guocity** [reply]
yea I installed the certificate, and all websites across all browsers are not working, some other app might work

**Reply by guocity** [reply]
> It seems the Proxyman Certificate is expired? Or you're using any VPN at the same time 🤔
> 
> You can try to factory reset in the Help menu -> Advance -> Factory Reset: Uncheck the "Include your Data"

other proxy work, but not proxyman

**Reply by guocity** [reply]
> It seems the Proxyman Certificate is expired? Or you're using any VPN at the same time 🤔
> 
> You can try to factory reset in the Help menu -> Advance -> Factory Reset: Uncheck the "Include your Data"

<img width="326" alt="image" src="https://github.com/user-attachments/assets/86451ddb-b6be-4095-aaf9-774c2bf91ab1">

**Reply by guocity** [reply]
<img width="319" alt="image" src="https://github.com/user-attachments/assets/59bebf35-9139-4fe9-89f7-7b24da5495a8">
if I disable Chrome setting: Block insecure private network requests. still google, apple, bing have same issue

>

**Reply by NghiaTranUIT** [reply]
This means that Proxyman can't open the SSL connection. Can you verify the Certificate is installed? You can find it in the Certificate menu -> Install for macOS 
 
<img width="812" alt="Screenshot 2024-09-08 at 19 41 37" src="https://github.com/user-attachments/assets/33ea96e3-2922-4982-aaaa-a274b7b4340a">

---------
May I ask if you're using any Security Endpoint?
